src/components/Navbar.js

```diff
@@ -1,32 +1,41 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
-
-export default function Navbar() {
-  const links = [
-    {
-      id: 1,
-      path: '/',
-      text: 'Book',
-    },
-    {
-      id: 2,
-      path: '/categories',
-      text: 'Categories',
-    },
-  ];
+import { Link } from 'react-router-dom';
+import '../styles/Navbar.css';
+import userImg from '../Assets/user.svg';
 
+function Navbar() {
   return (
-    <header>
-      <h1 className="title">Bookstore CMS</h1>
-      <nav className="na">
-        <ul className="nav-links">
-          {links.map((link) => (
-            <li key={link.id}>
-              <NavLink to={link.path}>{link.text}</NavLink>
-            </li>
-          ))}
+    <nav className="navbar">
+      <div className="left-side">
+        <h2 className="website-logo-name">Bookstore CMS</h2>
+        <ul>
+          <li>
+            <Link
+              className="links"
+              activeClassName="activelinks"
+              key="1"
+              to="/"
+            >
+              Books
+            </Link>
+          </li>
+          <li>
+            <Link
+              className="links"
+              activeClassName="activelinks"
+              key="2"
+              to="/categories"
+            >
+              Categories
+            </Link>
```

Comment:
- [ ] Currently, the active page should be indicated with a bold font-weight as on [Zeplin](https://zeplin.io/) design


![Screenshot](https://user-images.githubusercontent.com/57562869/156876324-d9eb1021-4bd2-4da7-be8f-04776e6e77b8.png)

Reply:
Cheers! Done.